rust/main/config/mainnet_config.json

```diff
@@ -2821,6 +2822,7 @@
       "name": "solanamainnet",
       "nativeToken": {
         "decimals": 9,
+        "denom": "lamport",
```

Comment:
if denom is not provided, it defaults to "" - what happens in this case?

ie is there now an expectation that all sealevel nativeToken metadata now includes `denom` too?

Reply:
afaict this field isn't used anywhere in agent logic, but is here just to be stored in the scraper DB along with the other data. So if it's empty and the chain isn't scraped there is no implication, otherwise the DB column will just be the empty string - but this assumption already existed before this PR

Reply:
yeah if possible I think we should try to keep the most succinct config surface area - it seems like it was provided here for completeness but isn't necessary? Imo just requiring `denom` for Cosmos is most natural and having it for Sealevel chains too could create some confusion as to what exactly needs to be configured when adding a Sealevel chain. So if possible I'd probably prefer to just keep denom as a Cosmos thing, possibly even making it an Option<String> for the new `NativeToken` that isn't Cosmos-specific

Reply:
Field `nativeToken.denom` is not used in Scraper logic for Sealevel chain, so, we can default it to any value. 

For Sealevel chains, though, the denomination depends on the chain itself: Solana - `lamport`, Eclipse - `gwei`.

Reply:
I added defaults so that we don't need to add `nativeToken` to the configuration.